chore: improve tools description

oweid opened the pull request.

Commit:
chore: improve tools description

Commit:
fix: missing punctuation

Commit:
chore: update en/us version

Commit:
chore: create chinese version of releasearchive.md

Commit:
fix: revert commit

I apologize for my mistake that I added the commit to the wrong branch.

ghost commented:
Regarding the development of the built-in ShadowsocksR for V2Ray, I would like to provide an update to the esteemed developers. 
Last Sunday and yesterday, I completely refactored the code and designed a simple plugin interface. 
This afternoon, ~~with no additional classes at school,~~ I have some available time and plan to complete the main body of the code. I believe the workload should not be substantial, as it is merely the addition of an interface—the core functionality of ShadowsocksR lies in its plugins. I will allocate time later to develop the subsequent plugins, and I also consider that the ShadowsocksRR plugin can be incorporated at a later stage.

Thank you for your attention.

AkinoKaede commented on docs/en_US/awesome/tools.md:
```diff
@@ -6,42 +6,75 @@
+/>
+
+<Tool
+    url="https://github.com/2dust/v2rayNG"
```
v2rayNG is currently not using v2ray-core.

AkinoKaede commented on docs/en_US/awesome/tools.md:
```diff
@@ -6,42 +6,75 @@
 <Tool
-    url="https://github.com/yanue/V2rayU"
-    name="V2rayU"
+    url="https://github.com/yanue/V2rayU/tree/master"
```
I’m not sure if V2RayU is still actively being updated. It appears to have been deleted.

Commit:
Chore: Remove v2rayNG

Remove v2rayNG as it is currently not using v2ray-core any more.

Commit:
Chore: Mark v2rayU as not updating

V2rayU has already set the branch "rm" as the main branch, which means that the developer is not maintaining the repository any more.

Commit:
Chore: Update English version

Commit:
Chore: fix type error

Commit:
fix: revert error

Commit:
Chore: Mark nekoray as stopped maintaining

AkinoKaede commented on docs/awesome/tools.md:
```diff
@@ -40,33 +40,27 @@
     url="https://github.com/yanue/V2rayU/tree/master"
```
I had unpacked v2rayU, but it’s not currently using v2ray-core.

Commit:
Chore: remove v2rayU

Not using v2ray-core.

Commit:
Chore: Update en/us version

AkinoKaede commented:
Thanks for your contribution.
